sig-scalability/charter.md

```diff
@@ -0,0 +1,110 @@
+performance improvements (that don't fall into the charter of another individual
+SIG) by driving large architectural changes and finding bottlenecks, as well as
+provide consultations about any scalability and performance related aspects of
+Kubernetes.
+
+### In Scope
+
+#### Code, Binaries and Services:
+
+- Scalability and performance testing frameworks. Examples include:
+  - [Cluster loader](https://github.com/kubernetes/perf-tests/tree/master/clusterloader2)
+  - [Kubemark](https://github.com/kubernetes/kubernetes/tree/master/cmd/kubemark)
+- Scalability and performance tests:
+  - [Tests](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/)
+
+#### Cross-cutting and Externally Facing Processes
+
+- Defining what does “Kubernetes scales” mean by defining (or approving)
+individual performance SLIs/SLOs, ensuring they are all oriented on user
+experience and consistent with each other:
+  - [SLIs/SLOs](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md)
+- Ensuring that each official Kubernetes release satisfies all scalability and
+performance related requirements, as stated in "Kubernetes scalability" definition.
+- Establishing and documenting best practises on how to design and/or implement
+Kubernetes features in scalable and performant way. Educating contributors and
+consulting individual designs/implementations to ensure that those are widely used.
+Example artifacts:
+  - [Scalability governance](https://github.com/kubernetes/community/blob/master/sig-scalability/governance)
+- Finding system bottlenecks and coordinating improvement on cross-cutting
+architectural changes.
+
+### Out of scope
+
+- Improving performance/scalability of features falling into charters of
+individual SIGs.
+
+
+## What can we do/require from other SIGs
+Scalability and performance are horizontal aspects of the system - changes in a
+single place of Kubernetes may affect the whole system. As a result, to
+effectively ensure Kubernetes scales, we need a special cross-SIG privileges.
+
+- We can rollback any merged PR if it has been identified as a cause of any
+  [performance/scalability SLOs] regression. The offending PR should only be
+  merged again after proving to pass  tests at scale.
+- We can pause the merge queue in case of a regression observed until a particular
+  PR has been identified as cause of the regression and regression has been
+  mitigated. The “Rules of engagement” of pausing merge-queue and rationale for
+  necessity of its introduce are explained in a separate doc. <br/>
+  TODO(wojtek-t, shyamjvs): Write it down and link here.
+- We require significant changes (in terms of impact, such as: update of etcd,
+  update of Go version, major architectural changes, etc.) may only be merged:
+  - with an explicit approval from a [SIG-scalability approver](#sig-scalability-approvers)
+    and
+  - after having passed performance testing on biggest supported clusters (unless
```

Comment:
In a future where there are large cluster jobs being run on multiple cloud providers, will you require that they all pass?

How does someone run these jobs as a presubmit?